invenio_records_lom/fixtures/demo.py

```diff
@@ -0,0 +1,363 @@
+    }
+
+
+def create_fake_lifecycle(fake: Faker) -> dict:
+    """Create a fake "lifeCycle"-element, compatible with LOMv1.0-standard."""
+    roles = [
+        "author",
+        "publisher",
+        "unknown",
+        "initiator",
+        "terminator",
+        "validator",
+        "editor",
+        "graphical designer",
+        "technical implementer",
+        "content provider",
+        "technical validator",
+        "educational validator",
+        "script writer",
+        "instructional designer",
+        "subject matter expert",
+    ]
+
+    statuses = ["draft", "final", "revised", "unavailable"]
+
+    random_int = fake.random.randint
+
+    return {
+        "version": langstringify(fake, f"{random_int(0,9)}.{random_int(0,9)}"),
+        "status": vocabularify(fake, statuses),
+        "contribute": [create_fake_contribute(fake, roles) for __ in range(2)],
+    }
+
+
+def create_fake_metametadata(fake: Faker) -> dict:
+    """Create a fake "metaMetadata"-element, compatible with LOMv1.0-standard."""
+    roles = ["creator", "validator"]
+    return {
+        "identifier": [create_fake_identifier(fake) for __ in range(2)],
+        "contribute": [create_fake_contribute(fake, roles) for __ in range(2)],
+        "metadataSchemas": ["LOMv1.0"],
+        "language": create_fake_language(fake),
+    }
+
+
+def create_fake_technical(fake: Faker) -> dict:
+    """Create a fake "technical"-element, compatible with LOMv1.0-standard."""
+    return {
+        "format": [fake.random.choice([fake.mime_type(), "non-digital"])],
+        "size": str(fake.random.randint(1, 2 ** 32)),
+        "location": [
+            fake.uri(),
+        ],
+        "requirement": [create_fake_requirement(fake) for __ in range(2)],
+        "installationRemarks": langstringify(fake, fake.paragraph()),
+        "otherPlatformRequirements": langstringify(fake, fake.paragraph()),
+    }
+
+
+def create_fake_requirement(fake: Faker) -> dict:
+    """Create a fake "requirement"-element, compatible with LOMv1.0-standard."""
+    return {
+        "orComposite": [create_fake_orcomposite(fake) for __ in range(2)],
+    }
+
+
+def create_fake_orcomposite(fake: Faker) -> dict:
+    """Create a fake "orComposite"-element, compatible with LOMv1.0-standard."""
+    type_ = fake.random.choice(["operating system", "browser"])
+    if type_ == "operating system":
+        requirement_names = [
+            "pc-dos",
+            "ms-windows",
+            "macos",
+            "unix",
+            "multi-os",
+            "none",
+        ]
+    else:
+        requirement_names = [
+            "any",
+            "netscape communicator",
+            "ms-internet explorer",
+            "opera",
+            "amaya",
+        ]
+
+    return {
+        "type": vocabularify(fake, [type_]),
+        "name": vocabularify(fake, requirement_names),
+        "minimumVersion": str(fake.random.randint(1, 4)),
```

Comment:
Minor: is it really necessary to convert it to a string?

Reply:
If we're strictly adhering to the LOMv1.0 standard, then it is:
![image](https://user-images.githubusercontent.com/86989418/128829386-42adc6c8-6776-45c7-99d0-ca14a75d3e80.png)
This goes for all string calls, every simple value is required to be of datatype "CharacterString".